Sources/NostrSDK/NIP44v2Encrypting.swift

```diff
@@ -199,16 +199,21 @@ extension NIP44v2Encrypting {
 
         let combined = aad + message
 
-        return Data(HMAC<CryptoKit.SHA256>.authenticationCode(for: combined, using: SymmetricKey(data: key)).bytes)
+        return Data(CryptoKit.HMAC<CryptoKit.SHA256>.authenticationCode(for: combined, using: SymmetricKey(data: key)))
+    }
+
+    private func toBytes(from data: Data) -> [UInt8] {
```

Comment:
This is kind of gross, but I'm doing this intentionally because I was getting ambiguous definition clashes from importing the `CryptoSwift` library. It defines a `var bytes: [UInt8]` as an extension to `Data`, and it clashes with `bytes` from the `secp256k1` library. I chose to copy the implementation here locally and use this instead.